apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,74 @@
+import { Client } from "auth0";
+
+describe("Client API", () => {
```

Comment:
Do we have a test for ensuring that you can't create a new application if you already have one?

Reply:
I'll add one.